docs/reference.md

```diff
@@ -434,6 +434,14 @@ The following Composer packages are also available:
 - guzzlehttp/guzzle       v6.3.0
 - ramsey/uuid             v3.6.1
 
+## Ruby actions
+
+Ruby actions are executed using Ruby 2.5. To use this runtime, specify the `wsk` CLI parameter `--kind ruby:2.5` when creating or updating an action. This is the default when creating an action with file that has a `.rb` extension.
```

Comment:
You’ll have modified the cli then to recognize this extension. I have an alternate proposal here: https://github.com/apache/incubator-openwhisk-cli/issues/286

Reply:
Ok. I will keep my eyes open on it.